tests/test_examples.py

```diff
@@ -875,6 +878,113 @@ def test_only_non_eq_optimization():
     np.testing.assert_allclose(obj.compute(*obj.xs(surf)), 1, atol=1e-5)
 
 
+@pytest.mark.regression
+@pytest.mark.solve
+@pytest.mark.slow
+def test_freeb_vacuum():
+    """Test for free boundary vacuum stellarator."""
+    extcur = [4700.0, 1000.0]
+    ext_field = SplineMagneticField.from_mgrid(
+        "tests/inputs/mgrid_test.nc", extcur=extcur
+    )
+    surf = FourierRZToroidalSurface(
+        R_lmn=[0.70, 0.10],
+        modes_R=[[0, 0], [1, 0]],
+        Z_lmn=[-0.10],
+        modes_Z=[[-1, 0]],
+        NFP=5,
+    )
+
+    eq = Equilibrium(M=6, N=6, Psi=-0.035, surface=surf)
+    eq.solve()
+    constraints = (
+        ForceBalance(eq=eq),
+        FixCurrent(eq=eq),
+        FixPressure(eq=eq),
+        FixPsi(eq=eq),
+    )
+    objective = ObjectiveFunction(VacuumBoundaryError(eq=eq, ext_field=ext_field))
+    eq, out = eq.optimize(
+        objective,
+        constraints,
+        optimizer="proximal-lsq-exact",
+        verbose=3,
+        options={},
+    )
+    rho_err, _ = area_difference_vmec(eq, "tests/inputs/wout_test_freeb.nc")
+
+    np.testing.assert_allclose(rho_err[:, -1], 0, atol=4e-2)  # only check rho=1
+
+
+@pytest.mark.regression
+@pytest.mark.solve
+@pytest.mark.slow
+def test_freeb_axisym():
+    """Test for free boundary finite beta tokamak."""
+    extcur = [
+        3.884526409876309e06,
+        -2.935577123737952e05,
+        -1.734851853677043e04,
+        6.002137016973160e04,
+        6.002540940490887e04,
+        -1.734993103183817e04,
+        -2.935531536308510e05,
+        -3.560639108717275e05,
+        -6.588434719283084e04,
+        -1.154387774712987e04,
+        -1.153546510755219e04,
+        -6.588300858364606e04,
+        -3.560589388468855e05,
+    ]
+    ext_field = SplineMagneticField.from_mgrid(
+        r"tests/inputs/mgrid_solovev.nc", extcur=extcur
+    )
```

Comment:
The `raw_coil_cur` array in the mgrid file itself is all ones, as is normal for raw mode

Reply:
These are the currents from the VMEC input which gives the correct field and boundary shape. This is why I was concerned about the raw vs scaled stuff, because even though this one is "raw" the extcur is still needed to get the right field

Reply:
I mean these were the values in the VMEC input that came with the mgrid file, I think from jonathan.